### DISCUSSION TITLE
Public link only latest image [SOLVED]

### DISCUSSION THREAD

**Discussion by evripidis** [aaronwmorris/indi-allsky#962] [General]
Greetings!

Is it possible to have/use a public link only with the latest image, without the sidebar?

I am thinking of adding my camera to a weather website and I do not want to have the ability to use the side bar.

**Comment by aaronwmorris** [comment]
The file at `https://hostname/indi-allsky/images/latest.jpg` should always be available.

You can also make a call to `https://hostname/indi-allsky/js/latest` and it will return JSON with a URL to the latest image.

**Reply by evripidis** [reply]
That's exactly what I was looking for! Thank you!
